<?xml version="1.0"?>
<launch>
    <node pkg="tl_detector" type="tl_detector.py" name="tl_detector" output="log" cwd="node">
    	<param name="use_image_clips" value="False" />
        <param name="use_image_array" value="False" />
        <param name="save_images" value="False" />
    </node>
</launch>
